xml/dm_registers.xml

```diff
@@ -221,8 +221,10 @@
         <field name="dataaccess" bits="16" access="R" reset="Preset">
             0: The {\tt data} registers are shadowed in the hart by CSR
             registers. Each CSR register is XLEN bits in size, and corresponds
-            to a single argument, per Table~\ref{tab:datareg}.
-
+            to a single argument, per Table~\ref{tab:datareg}. These
+	    implementation-specific CSRs should be located in the ``Non-standard''
```

Comment:
It looks like these lines are indented less than the surrounding block.

I would add parens indicating which CSRs are the non-standard range. That saves somebody 5 minutes of digging, and is unlikely to change.

Reply:
@aswaterman These are optional, and I was thinking we didn't need to standarize their location (since the whole point of this register is to describe where they are).

After thinking about this more, it's really only needs a max of 3 CSRs (I was thinking it would be up to 12, but these CSRs should be XLEN wide, not 32). So I think we should just standardize that if you're going to do it this way its the last 3 Debug CSRs which are XLEN bits wide. Which would render this field in hartinfo sort of pointless.

I'm not sure how you would use this information when the CSR width can change based on writable $misa. But this is already unclear in the spec as written -- how many CSRs do you actually need to access all the $dataN registers.

Will fix the indentation

Reply:
Ah, I realized that it does say each CSR corresponds to a single argument, so it is pretty clear how that corresponds to XLEN-wide CSRs.